README.md

```diff
@@ -10,6 +10,7 @@ This app uses the json api provided by a server maintained [here](https://github
 - The app on startup picks up data from a json file in it's assets folder if the version number of data is -1 which happens when there is no internet connection
 - If there is a valid internet connection, the data download starts from the server.
 - Also there is a check on the version of data already there in the app's database. If data is stale then only it is downloaded.
+<<<<<<< HEAD
```

Comment:
@creativepsyco There were lot of merge conflicts while rebasing but I resolved them. Seems like I forgot this HEAD arrow. I'll remove it.